mantis/analysis/settings/example_concatenate_datasets.yml

```diff
@@ -0,0 +1,6 @@
+concat_data_paths: ["/path/to/data.zarr/*/*/*","/path/to/data2.zarr/*/*/*"]
+time_indices: all
+channel_names: [all,all]
+X_slice: [0, 100]
+Y_slice: [0, 100]
+Z_slice: [0, 100]
```

Comment:
If we decide to keep these, an `all` setting would be helpful.

Reply:
I made them optional by making them 'all'. One can remove them from the example yml file for now if needed.